src/models/order.js

```diff
@@ -8,24 +8,27 @@ module.exports = (sequelize, DataTypes) => {
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
-      Order.belongsTo(models.User);
-      Order.belongsTo(models.Team);
+      Order.belongsTo(models.User, {
+        foreignKey: 'userId',
+        allowNull: false,
+      });
+      Order.belongsTo(models.Team, {
+        foreignKey: 'teamId',
+        allowNull: false,
+      });
     }
   }
   Order.init(
     {
-      orderNumber: {
-        type: DataTypes.INTEGER,
-        allowNull: false,
-        unique: true,
-      },
-      userId: {
+      id: {
         type: DataTypes.INTEGER,
-        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
       },
-      teamId: {
+      orderNumber: {
```

Comment:
I think it makes more sense to just make the `orderNumber` the primary key - wdyt?

Reply:
That's fine with me - I think using a natural key as our PK probably works better in this case.